src/DTO/OrderStateInstallerData.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace Mollie\DTO;
+
+class OrderStateInstallerData
```

Comment:
OrderStateData IMO, it's not installer data.

Reply:
renamed